djangocms_blog/sitemaps/__init__.py

```diff
@@ -3,6 +3,7 @@
 
 from cms.utils import get_language_list
 from django.contrib.sitemaps import Sitemap
+from django.urls.exceptions import NoReverseMatch
```

Comment:
This import is valid in Django 1.10 only, you can try / except and add ``from django.conf.urls.exceptions import NoReverseMatch`` for previous versions